hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenUnfreezeAccountHandlerTest.java

```diff
@@ -185,6 +188,8 @@ void accountNotPresentInTxnBody() {
         @Test
         void tokenMetaNotFound() {
             final var token = MISSING_TOKEN_97531;
+            given(tokenStore.get(token))
+                    .willReturn(Token.newBuilder().tokenId(token).build());
```

Comment:
Kind of weird to have a test case where the token exists and the token meta doesn't 🤔 . Dunno 🤷‍♂️ just a thought